source/Radar.cpp

```diff
@@ -51,7 +52,9 @@ void Radar::SetCenter(const Point &center)
 }
 
 
-
+// Planets and stars both are circles, so they have "inner" as 0, but I can't
+// tell where it sets that. This may be clue to find where the identity of an object
+// is identified as a star.
```

Comment:
This isn't really the place for your notes.

Reply:
Already gone. Thanks!